.github/workflows/lattice-controller.yml

```diff
@@ -0,0 +1,96 @@
+name: LATTICE-CONTROLLER
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+    - "lattice-controller/**"
+    tags:
+    - 'lattice-controller-v*'
+  pull_request:
+    branches: [ main ]
+    paths:
+    - "lattice-controller/**"
+
+env:
+  CARGO_TERM_COLOR: always
+  working-directory: ./provider
```

Comment:
@autodidaddict here's the offender for the `file not found`, should be `working-directory: ./lattice-controller`